### ISSUE TITLE
Beginner HTML & CSS: Translate Getting Setup from English to Italian

### ISSUE THREAD

**Issue by PhilipHarney** [CoderDojo-Content/content-hack#13]
**If this is your first time working on *Beginner HTML & CSS* then you will need to be added as a collaborator. You can find instructions on how to do this at http://dojo.soy/gitbook-collab**
1. Make sure you have assigned this task card to yourself and moved it to the **In Progress** column.
2. Go to https://www.gitbook.com/book/coderdojo/beginner-html-css/edit and create a change request named **Translation of Getting Setup (README.md) to Italian**. See how to do this at http://dojo.soy/gitbook-makecr
3. Switch the view to Italiano it. See how to do this at http://dojo.soy/gitbook-changelang
4. Translate the English text into Italian. For an overview of what to translate, see http://dojo.soy/translation-guidelines.
5. Once the translation is complete, save your work by clicking the **Publish** button.
6. Move this task card into the **Review** column.
7. If you have time, pick another task from the **Next** column!

**Comment by fpiantini** [comment]
il would be nice to have a trinket start page in italian, like this one:
![Selection_060.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a08c0b7d4a0fa0011d682bb/125516c66e82c728ace21e0d46db9196268197a080e78013f20db2d86b56357c2301a03f32dfa17dbe137857575b1ca9480c0a10fbf98829afe269268f545dffcd7f1d2598694ef1e1c84afd13b0328a488fb7b98b4ec1ee3426ee8e8e4d3b0c48.png)
